src/classes/groundplaces.ts

```diff
@@ -15,6 +20,14 @@ import {
  * @description GroundPlaces business logic.
  */
 export class GroundPlaces {
+  private readonly Generator: Generator = new Generator();
+
+  public readonly Storage: Storage | null = null;
+
+  constructor(groundPlaces: GroundPlacesList) {
+    this.Storage = new Storage(groundPlaces);
```

Comment:
this.Storage (never use a capital for a property I think. @rimiti  ?

Reply:
yes, capitalize only fo class names, we often use smtg like `storageService` and  `generatorService`

Reply:
Done!